index.js

```diff
@@ -37,4 +38,36 @@ WPCOM_VIP.prototype.makeUrl = function( url ) {
     return this.req.makeUrl( url );
 };
 
+// Caps
+const CAP_API_READ = 1;
+const CAP_API_EDIT = 2;
+const CAP_API_ADD = 4;
+const CAP_API_DELETE = 8;
+
+WPCOM_VIP.prototype.currentUserCan = function( cap, action ) {
+    this.caps.forEach( function( c ) {
+        if ( c.resource_name == cap ) {
+            return action <= c.permissions;
```

Comment:
The return here won't actually re turn from the `currentUserCan` func - only the `forEach` one.

Instead we could do `return this.caps.some( function( c ) {...} )`.